app/package.json

```diff
@@ -3,46 +3,67 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "build": "next build",
-    "dev": "next dev",
+    "build": "NEXT_TELEMETRY_DISABLED=1 next build",
+    "dev": "NEXT_TELEMETRY_DISABLED=1 next dev",
     "format": "prettier --write '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "format-check": "prettier --check '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
-    "lint": "next lint",
-    "start": "next start",
+    "lint": "NEXT_TELEMETRY_DISABLED=1 next lint",
+    "postinstall": "mkdir ./public && cp -R ./node_modules/@uswds/uswds/dist/ ./public/uswds/",
+    "start": "NEXT_TELEMETRY_DISABLED=1 next start",
+    "storybook": "start-storybook -p 6006",
+    "storybook-build": "build-storybook",
     "test": "jest --ci --coverage",
     "ts:check": "tsc --noEmit"
   },
   "dependencies": {
-    "@types/jest-axe": "^3.5.3",
-    "@typescript-eslint/parser": "^5.26.0",
-    "jest-axe": "^6.0.0",
+    "@uswds/uswds": "^3.0.1",
     "next": "12.1.6",
     "react": "18.1.0",
-    "react-dom": "18.1.0"
+    "react-dom": "18.1.0",
+    "sass": "^1.52.3"
   },
   "devDependencies": {
+    "@babel/core": "^7.18.0",
     "@babel/preset-typescript": "^7.17.12",
+    "@storybook/addon-actions": "^6.5.9",
+    "@storybook/addon-essentials": "^6.5.9",
+    "@storybook/addon-interactions": "^6.5.9",
+    "@storybook/addon-links": "^6.5.9",
+    "@storybook/builder-webpack5": "^6.5.9",
+    "@storybook/manager-webpack5": "^6.5.9",
+    "@storybook/react": "^6.5.9",
+    "@storybook/testing-library": "^0.0.11",
```

Comment:
Similar comment here – this might be a dependency that should be opt-in rather than opt-out. It could be hard for a project team to know what they need to get the basics up and running versus what they're not using. I'm thinking of things like managing dependabot and security vulnerability notifications.

Reply:
> this might be a dependency that should be opt-in rather than opt-out

Is this comment specifically for `@storybook/testing-library` or for all four of these shown in the snippet?

Reply:
Ah, dang it GitHub. This comment was just on the testing library package.